src/custom-components/refComponents.ts

```diff
@@ -0,0 +1 @@
+export const ComponentWithRefs = ['Input']
```

Comment:
Maybe you need a dictionary to store html types, otherwise you will get type errors
example for Input, type is HTMLInputElement

in oc you will declare it as RefObject&lt;HTMLInputElement&rt;
and in logic you will create it as useRef&lt;HTMLInputElement&rt;

Reply:
done!